.github/workflows/release.yaml

```diff
@@ -57,7 +57,7 @@ jobs:
         uses: goreleaser/goreleaser-action@8f67e590f2d095516493f017008adc464e63adb1 # v4.1.0
         with:
           version: latest
-          args: release --rm-dist
+          args: release --rm-dist --timeout 60m
```

Comment:
i would add 120m just to be sure :)

Reply:
we need to add a COSIGN_YES env variable. I'll do it.

Reply:
https://github.com/sigstore/policy-controller/pull/540

Reply:
👍🏻